spec/fixtures/complete_case_log.json

```diff
@@ -126,7 +126,6 @@
     "gdpr_declined": "",
     "property_owner_organisation": "",
     "property_manager_organisation": "",
-    "sale_or_letting": "",
```

Comment:
Not sure if this is correct, as sale_or_letting might still be required for bulk uploads?

Reply:
Same